docs/using-connector.md

```diff
@@ -533,6 +534,160 @@ conf.set("spark.riakts.write.bulk-size", "500")
+ 
+ ```
+ /path/to/bin/kafka-console-producer.sh --broker-list localhost:9092 --topic streaming
+ ```
+ 
+ This will read a message from standard in and pass it to the topic. As an example put the following into standard in:
+ 
+ ```
+ {"time": "2016-01-01 08:30:00.000", "weather": "sunny", "temperature": 25.0, "humidity": 67.0, "pressure": 30.20, "family": "f"}
+ ```
+ 
+You should now be able to see this data entry in the KV bucket `test-data`.
+
+###Spark Streaming TS Table Example
+
+Having seen how spark streaming works with KV buckets, lets now look at the TS table example [here](../examples/src/main/scala/com/basho/riak/spark/examples/streaming/StreamingTSExample.scala). 
+
+The code is somewhat similar to the KV bucket example, but with crucial differences. Let have a look:
+
+```scala
+ val schema = StructType(List(
+      StructField(name = "weather", dataType = StringType),
+      StructField(name = "family", dataType = StringType),
+      StructField(name = "time", dataType = TimestampType),
+      StructField(name = "temperature", dataType = DoubleType),
+      StructField(name = "humidity", dataType = DoubleType),
+      StructField(name = "pressure", dataType = DoubleType)))
+
+    val sparkConf = new SparkConf(true)
+      .setAppName("Simple Spark Streaming to Riak TS Demo")
+
+    setSparkOpt(sparkConf, "spark.master", "local")
+    setSparkOpt(sparkConf, "spark.riak.connection.host", "127.0.0.1:8087")
+    setSparkOpt(sparkConf, "kafka.broker", "127.0.0.1:9092")
+
+    val sc = new SparkContext(sparkConf)
+    val streamCtx = new StreamingContext(sc, Durations.seconds(15))
+    val sqlContext = new org.apache.spark.sql.SQLContext(sc)
+    import sqlContext.implicits._
+
+    val kafkaProps = Map[String, String](
+      "metadata.broker.list" -> sparkConf.get("kafka.broker"),
+      "client.id" -> UUID.randomUUID().toString
+    )
+```
+
+This first section of code just sets up the table schema, a spark streaming context, a spark sql context, and Kafka properties. Note that we need to set up a table in TS that reflect the schema in the previous section of code. We can create this table in TS with:
+
+```
+curl -v -XPUT -H 'Content-Type: application/json' "http://$RIAK_HTTP/admin/explore/clusters/default/bucket_types/ts_weather_demo" -d '{"props":{"n_val":3, "table_def":"CREATE TABLE ts_weather_demo (weather varchar not null,family varchar not null,time timestamp not null,temperature  double,humidity double,pressure double,PRIMARY KEY ((weather, family, quantum(time, 1, 'h')), weather, family, time))"}}'
+```
+
+Be sure to substitute the Riak's IP address and http port in for `$RIAK_HTTP`.
```

Comment:
change "the Riak's IP address and http port..." to "the Riak node's IP address and HTTP port"